src/chat/commands/so.ts

```diff
@@ -0,0 +1,36 @@
+import { OnCommandEvent, OnSayEvent } from "../../models"
+import { EventBus, Events } from "../../events"
+import { ShouldThrottle } from '../shouldThrottle'
+
+/**
+ * Sends a message to chat with info on BBB Instagram
+ * @param onCommandEvent 
+ */
+export function So(onCommandEvent: OnCommandEvent): void {
+
+  const cooldownSeconds = 300
+
+  // The broadcaster is allowed to bypass throttling. Otherwise,
+  // only proceed if the command hasn't been used within the cooldown.
+  if (!onCommandEvent.flags.broadcaster &&
+    ShouldThrottle(onCommandEvent.extra.sinceLastCommand, cooldownSeconds, true)) {
+    return
+  }
+
+  // Only mods or broadcasters can call the shout-out command.
+  if (!onCommandEvent.flags.broadcaster &&
```

Comment:
Can we do this check first? That way we can exit fast if you're not a broadcaster or mod.